_posts/blog/2019-08-19-Ruslan's-Final-Report.md

```diff
@@ -0,0 +1,82 @@
+---
+layout: post
+title: GSoC Final Report by Ruslan 
+categories: blog
+excerpt: work product submission on the scona project during GSoC program
```

Comment:
Do you mind if we change this? I think this excerpt will make more people want to read it when they're scanning through the blog!
```suggestion
excerpt: Final report for work on the scona project during GSoC 2019
```

Reply:
Sure, I've changed that